wifiphisher/extensions/deauth.py

```diff
@@ -10,7 +10,7 @@
 import wifiphisher.common.constants as constants
 
 
-logger = logging.getLogger("wifiphisher.deauth")
+logger = logging.getLogger(__name__)
```

Comment:
in every module, we can just define logger as `logging.getLogger("__name__")` without hardcoding the module name

Reply:
I think I tested and it failed to add it to the log file but I may be wrong. Have you tested this?

Reply:
Yes. It's ok on my system. Does the log in `pywifiphisher.py` get printed?

Reply:
I have to test this version but when I tested last it would log fine on the main but all the other modules were not logged.

Reply:
The logging message looks like following on my system:
It looks fine for the deauth module on my system.
```
2017-09-20 08:07:58,581 - wifiphisher.pywifiphisher - INFO - Unblocking interfaces
2017-09-20 08:07:58,627 - wifiphisher.pywifiphisher - INFO - Killing all interfering processes
2017-09-20 08:07:58,627 - wifiphisher.pywifiphisher - INFO - Changing wlan0 MAC address to 00:00:00:3f:96:93
2017-09-20 08:08:07,870 - wifiphisher.pywifiphisher - INFO - Selecting Firmware Upgrade Page template
2017-09-20 08:08:15,341 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
2017-09-20 08:08:15,931 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
2017-09-20 08:08:20,933 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
2017-09-20 08:08:21,057 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
2017-09-20 08:08:25,111 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
2017-09-20 08:08:25,159 - wifiphisher.extensions.deauth - WARNING - An error happened while extracting the packet's channel
```